src/Sulu/Bundle/ContentBundle/Preview/PreviewException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle\Preview;
+
+/**
+ * PreviewException
+ */
+class PreviewException extends \Exception
+{
+    /**
+     * PreviewException constructor.
+     *
+     * @param string $message
+     * @param int $code
+     * @param \Exception $previous
+     */
+    public function __construct($message = '', $code = 0, \Exception $previous = null)
+    {
+        parent::__construct($message, $code, $previous);
+    }
```

Comment:
I would not override the constructor if you just call the parent constructor with the same arguments.